doc/source/dev/core-dev/distributing.rst.inc

```diff
@@ -90,25 +109,19 @@ Supported Python and NumPy versions
 The Python_ versions that SciPy supports are listed in the list of PyPI
 classifiers in ``setup.py``, and mentioned in the release notes for each
 release.  All newly released Python versions will be supported as soon as
-possible.  The general policy on dropping support for a Python version is that
-(a) usage of that version has to be quite low (say <5% of users) and (b) the
-version isn't included in an active long-term support release of one of the
-main Linux distributions anymore.  SciPy typically follows NumPy, which has a
-similar policy.  The final decision on dropping support is always taken on the
-scipy-dev mailing list.
+possible.  For the general policy on dropping support for a Python or NumPy
+version, see :ref:`NEP 29 <NEP29>`.  The final decision on dropping support is
+always taken on the scipy-dev mailing list.
 
 The lowest supported Numpy_ version for a SciPy version is mentioned in the
-release notes and is encoded in ``scipy/__init__.py`` and the
+release notes and is encoded in ``pyproject.toml``, ``scipy/__init__.py`` and the
 ``install_requires`` field of ``setup.py``.  Typically the latest SciPy release
-supports 3 or 4 minor versions of NumPy.  That may become more if the frequency
-of NumPy releases increases (it's about 1x/year at the time of writing).
-Support for a particular NumPy version is typically dropped if (a) that NumPy
-version is several years old, and (b) the maintenance cost of keeping support
-is starting to outweigh the benefits.  The final decision on dropping support
-is always taken on the scipy-dev mailing list.
-
-Supported versions of optional dependencies and compilers is less clearly
-documented, and also isn't tested well or at all by SciPy's Continuous
+supports ~5 minor versions of NumPy (i.e. up to 2.5 years' old NumPy versions,
```

Comment:
Maybe drop the sentence entirely since it is covered above (current numpy + 2 or 3)? The actual number is dependent on oldest-supported-numpy (which is not mentioned here I think) and can vary per platform.

Reply:
Nope, it's not covered by `oldest-supported-numpy`, that's a different scheme, explicitly does not follow NEP 29. I will explained `oldest-supported-numpy` in the NumPy doc PR following this one. SciPy doesn't use `oldest-supported-numpy` for now (see gh-13520).

I'll look at removing the sentence if it's duplicate.

Reply:
Extended the sentence a bit instead, to be more clear.